content/en/docs/reference/glossary/sig.md

```diff
@@ -2,7 +2,7 @@
 title: SIG (special interest group)
 id: sig
 date: 2018-04-12
-full_link: https://github.com/kubernetes/community/blob/master/sig-list.md#master-sig-list
+full_link: https://www.kubernetes.dev/community/community-groups/#special-interest-groups
```

Comment:
```suggestion
full_link: https://github.com/kubernetes/community/blob/master/sig-list.md#special-interest-groups
```
Imo, we should keep the full link to k/community and not to the contributor site as the source of truth about SIGs is k/community
Additionally we also link to the SIG governance in the glossary entry for SIGs which is also to the k/community repo

Reply:
They are the exactly the same file.
https://github.com/kubernetes/community/blob/11f8c35fa86ee29108503aca5532b994cda74926/sig-list.md?plain=1#L7-L10

Just like we won't use https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/sig.md as the source of truth but https://kubernetes.io/docs site.

Reply:
Does [k8s.dev](https://www.kubernetes.dev/) is built by [k/community](https://github.com/kubernetes/community) repo ? (like kubernetes.io built by this repo)
I checked that the github icon(at the bottom of the k8s.dev page) makes link to the k/community, but I'm not sure that the page sources were built based on k/commnunity repository.
It that's right, then I also think link to k8s.dev is better than k/community ! 😄

Reply:
k8s.dev is built by the k/contributor-site repo but the source of truth of sigs is k/community

Reply:
Hey @bconfiden2, maybe it's better to use k/community link for now.

The tiny issue is not necessary to block this. We can have discussion about k8s.io vs k/community after this PR merged.

Reply:
Fixed it again!! Thank you for discussion @reylejano @Sea-n 😃 😃